automation-tests/test/scratchOrg.ts

```diff
@@ -0,0 +1,317 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import child_process from 'child_process';
+import fs from 'fs';
+import path from 'path';
+import util from 'util';
+import {
+  DefaultTreeItem,
+  InputBox,
+  QuickOpenBox,
+  Workbench
+} from 'wdio-vscode-service';
+import {
+  EnvironmentSettings
+} from './environmentSettings';
+import {
+  utilities
+} from './utilities';
```

Comment:
We should probably at least have a ticket to get eslint/prettier integrated with the project it'll detect and fix things like imports out of order